game.js

```diff
@@ -68,10 +68,10 @@ var reset = function () {
   hero.x = canvas.width /2;
   hero.y = canvas.height /2 ;
   // Place the monster somewhere on the canvas randomly
-  
-  monster1.x = 32 + (Math.random() * (canvas.width - 64));
-  monster1.y = 32 + (Math.random() * (canvas.height - 64));
-  
+
+  monster1.x = 12 + (Math.random() * (canvas.width - 512));
```

Comment:
This function `Math.random()` generates a random number. You don't want to generate a random number. You want to set it at x = 0 and y = 0.